src/DayPicker.js

```diff
@@ -122,6 +122,7 @@ export default class DayPicker extends Component {
     canChangeMonth: true,
     reverseMonths: false,
     pagedNavigation: false,
+    modifiersStyles: {},
```

Comment:
`modifiersStyles` should have an `undefined` default value. This way we can avoid calculations when building the `style` object in the Day cell.